include/mbedtls/config.h

```diff
@@ -217,6 +217,8 @@
  * major version by making sure your code is not using these functions.
  *
  * Uncomment to get errors on using deprecated functions.
+ *
+ * Requires !MBEDTLS_MD2_C, !MBEDTLS_MD4_C, !MBEDTLS_RC4_C
```

Comment:
The meaning of exclamation marks are pretty straightforward for us, but they are easy to overlook and might leave a sense of doubt in the reader. I suggest we clarify this a little bit by writing "Requires to disable the following macros:"

Reply:
Done.